Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsFragmentPresenter.java

```diff
@@ -253,6 +253,7 @@ private void addGraphicsSettings(ArrayList<SettingsItem> sl)
 		}
 		else
 		{
+			mSettings.get(SettingsFile.SETTINGS_DOLPHIN).put(SettingsFile.SECTION_CORE, new SettingSection(SettingsFile.SECTION_CORE));
```

Comment:
Oh yeah, you're right. I figured it didn't matter if I just did it this way, but thinking about it more, putting it here has the potential to wipe settings if the graphics menu is opened after the main settings menu. I'll fix it when I get back to my computer.